### PR TITLE
[RNMobile] Use multiple sliders in column parent

### PR DESCRIPTION
**Pull request by lukewalczak**
## Description

That PR introduces multiple sliders in the column's parent to control width of all inner columns.

## How has this been tested?

1. Open mobile app
2. Add columns block
3. Press `Cancel` button
4. Open columns settings
5. Play with sliders
**6. Expect that fill level in the preview (to the left of a slider) is changing proportionally in all columns** 
 

## Types of changes

New feature

## Checklist:
- [x] My code is tested.
- [x] My code follows the WordPress code style. 
- [x] My code follows the accessibility standards. 
- [x] My code has proper inline documentation. 
- [x] I've included developer documentation if appropriate. 
- [x] I've updated all React Native files affected by any refactorings/renamings in this PR.

### CODE REVIEW

File: packages/block-library/src/columns/edit.native.js [hunk 1]
```diff
@@ -140,6 +144,36 @@ function ColumnsEditContainer( {
...
+					}
+					cellContainerStyle={ styles.cellContainerStyle }
+					toFixed={ 1 }
+					rangePreview={
+						<ColumnsPreview
+							columnWidths={ columnWidths }
+							selectedColumnIndex={ index }
+						/>
+					}
+					key={ column.id }
```
Comment (dratwas) [inline packages/block-library/src/columns/edit.native.js:170]:
```suggestion
					key={ column.clientId }
```

File: packages/components/src/mobile/bottom-sheet/range-cell.native.js [hunk 2]
```diff
@@ -228,20 +233,22 @@ class BottomSheetRangeCell extends Component {
...
-							{ width: 40 * fontScale },
-						] }
-						onChangeText={ this.handleChange }
-						onFocus={ this.handleToggleFocus }
-						onBlur={ this.handleToggleFocus }
-						keyboardType="numeric"
-						returnKeyType="done"
-						value={ `${ sliderValue }` }
-					/>
+					{ withTextInput && (
```
Comment (pinarol) [inline packages/components/src/mobile/bottom-sheet/range-cell.native.js:236]: isn't 'withXXX' is a naming convention for HOCs? This looks like a boolean prop so, can we update the naming to reflect the intent more?

**Review by pinarol — COMMENTED** [review]
Things seem to work as intended on iOS side. Just dropped a code style comment. Good job!

File: packages/components/src/mobile/bottom-sheet/range-cell.native.js (hunk 2 above)
Comment (lukewalczak) [reply packages/components/src/mobile/bottom-sheet/range-cell.native.js]: That's true, I've renamed it to `shouldDisplayTextInput`

**Review by dratwas — APPROVED** [review]

File: packages/components/src/mobile/bottom-sheet/range-cell.native.js (hunk 2 above)
Comment (pinarol) [reply packages/components/src/mobile/bottom-sheet/range-cell.native.js]: Thanks!

**Review by pinarol — APPROVED** [review]
LGTM!

**Pull request merged**